### DISCUSSION TITLE
need help on Analytics

### DISCUSSION THREAD

**Discussion by srujanmadderla** [outline/outline#6814] [General]
Hi Team,

Can someone please guide us how to track the productivity on outline wiki?

my team is creating knowledge article and i would need to pull the report of the users who are contributing,

Please guide

**Comment by srujanmadderla** [comment]
can anyone please guide on this ?

**Reply by tommoor** [reply]
You would need to read the `events` table in postgres and construct your report using events such as `revisions.create`, `documents.create`, and `documents.update`

**Reply by srujanmadderla** [reply]
Hello Tommoor,

Thank you for the response.
Could you please help me with the steps or any relevant screenshots ?

**Reply by tommoor** [reply]
I'm afraid this requires some sort of BI tooling to create reports from your database such as Metabase, Mode, etc. There is no functionality built into Outline to provide reports like this yet.
